### ISSUE TITLE
Consider verbage

### ISSUE THREAD

**Issue by ilias-t** [reconbot/mit-3-laws-license#2]
Not a lawyer, but trying to think like one... Seems it would be easier to enforce the modified clause:

"This software may not be used to **intentionally** injure a human being or, through inaction, allow a human being to come to harm"

_Injure_ and _harm_ would likely need to be more explicitly defined as well.

**Comment by pluma** [comment]
I think this is especially important with regard to "inaction". Otherwise you'd restrict use to software that is actually capable of preventing harm (rather than simply banning certain uses of software that is capable of causing harm).

**Comment by ilias-t** [comment]
Yeah inaction would likely need to be heavily qualified along the lines of “gross negligence”, because it would otherwise encompass too many hypotheticals.

**Comment by ilias-t** [comment]
Also I want to note the verbage isn’t a word, but verbiage is. Regardless, “Consider Language” would probably be a better title for this issue.
